ios/RNGestureHandlerModule.mm

```diff
@@ -173,6 +173,22 @@ - (void)setBridge:(RCTBridge *)bridge
     }];
 }
 
+RCT_EXPORT_METHOD(flushQueuedHandlers)
```

Comment:
I'm thinking of renaming this method to something like `flushGestureHandlerOperations` or `flushOperations` 🤔

Reply:
I went with [flushOperations](https://github.com/software-mansion/react-native-gesture-handler/pull/1979/commits/da1592520ebf7f94e9a060a68b5bbe83efc6d2a1), as this method will be called on `RNGestureHandlerModule` anyway. IMO it's self-explanatory that it relates to GH.